package.json

```diff
@@ -268,6 +278,7 @@
     "test-integration-db2": "cross-env DIALECT=db2 yarn test-integration",
     "test-integration-ibmi": "cross-env DIALECT=ibmi yarn test-integration",
     "test-integration-snowflake": "cross-env DIALECT=snowflake yarn test-integration",
+    "test-integration-yugabyte": "export DIALECT=yugabyte SEQ_TEST_CLEANUP_TIMEOUT=100000  && yarn mocha \"test/integration/**/*.test.[tj]s\" --timeout 600000",
```

Comment:
Here timeout is in milliseconds right? Whats the difference between 100000 and 600000 here? Is 100sec enough for test cleanup in case of yugabytedb?

Reply:
Yes, timeout is in milliseconds. `SEQ_TEST_CLEANUP_TIMEOUT ` is the database cleanup timeout after each test and `--timeout` is the overall each test should at max to execute. I think it should be enough I have tried running the whole suite with these timeouts  and i didn't face issues related to any cleanup timeout issues. I can increase it to 500k for max case if you want to me do that @sanyamsinghal ?

Reply:
its fine if you have tested it on small/medium sized machine on which database is running.